src/components/BookDetails/BookDetailsMain.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import BookAuthor from '../BookDetails/BookAuthor.jsx';
+import BookCover from '../BookDetails/BookCover.jsx';
+import BookDescription from '../BookDetails/BookDescription.jsx';
+import BookPublishDate from '../BookDetails/BookPublishDate.jsx';
+import BookQuote from '../BookDetails/BookQuote.jsx';
+import BookTitle from '../BookDetails/BookTitle.jsx';
+import styles from './BookDetailsMain.module.css';
+
+export default function BookDetailsMain({
+	title,
+	coverImg,
+	description,
+	publishDate,
+	quote,
+	author,
+}) {
+	return (
+		<div className={styles.bookDetailsContainer}>
+			<div className={styles.cover}>
+				<BookCover coverImg={coverImg} />
```

Comment:
These components are all one-liners and not worth re-using since their CSS is likely going to change in the future anyways. Let's keep all the HTML and CSS in this component.